### PR TITLE
Build/hamburger component

### PR DESCRIPTION
**Pull request by oacortez**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- Created a basic structure of our Navbar/Hamburger menu 
- Link all the paths for the user to go to 
- implemented basic css structure / animation for Navbar/hamburger menu
- Made our IMG a Link to home view
- Implemented a new Npm package called "classNames" - for animation purposes 

What (if anything) did you refactor?
- Remove a NPM package(Not classNames, but a different one) that I installed. Later on discovered that I did not use it
- Change the color of the Navbar to a new color (Can be changed later on - Only changed it due to view it cleaner) 

Were there any issues that arose?
~I have an issue where I can't fully make the width cover the whole page when the user clicks on the hamburger menu~
-I was not able to make it responsive
~When the animation opens, it creates a vertical scroll~
~Not sure if there's a lot of bugs in the CSS or if my component structure and classNames are causing problems those CSS bugs~

Is there anything that you need from your teammate?
- Would love for someone to view any mistakes I made, since my CSS knowledge is very poor and see if maybe we can fix some of these bugs that I ran into
- It was my first time creating something like this so some of the animation code I applied I am not familiar with

Any other comments, questions, or concerns?
- Is not perfect, but I think is decent enough for now. Definitely will need a good refactor. Hopefully not a huge refactor where it will have to start from scratch again 🙁
- Ryan: Made some adjustments to the styling and it seemed to fix the scroll problem. Also set the boolean value in state, which fixed the console errors

### CODE REVIEW

**Review by Echory — APPROVED** [review]
Looks really good, I like the small version of the menu!

**Review by aliroemhildt — APPROVED** [review]
Saw the drop down in action and it looks awesome! Thanks Oscar and Ryan for your work on this

File: src/components/HamburgerMenu/HamburgerMenu.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <div className={displayMobileNavLinks}>
+      <Link to="/" className="mobile-link" onClick={() => setMobileNavOpen(!open)}>Home</Link>
+      <Link to="/submission" className="mobile-link" onClick={() => setMobileNavOpen(!open)}>Submission Form</Link>
+      <Link to="/admin" className="mobile-link" onClick={() => setMobileNavOpen(!open)}>Admin</Link>
+    </div>
+  )
+}
+
+export default HamburgerMenu;
```
Comment (mr-ryan12) [inline src/components/HamburgerMenu/HamburgerMenu.js:21]: Thank you for doing this Oscar. I made a couple changes. I removed the buttons nested inside the <Link> tags and that seemed to fix some of the issues. Please reach out if you have any questions!

**Review by mr-ryan12 — APPROVED** [review]
Many thanks! Looks great!

**Pull request merged**